index-advisor/getting-started.mdx

```diff
@@ -65,6 +65,20 @@ for warnings in the collector test output (`pganalyze-collector --test
 --verbose`).
 
 
+## Set up the extended stats helper for better recommendations
+
+Extended statistics are used by pganalyze to improve Index Advisor
```

Comment:
This might be a good place to link to https://www.postgresql.org/docs/current/sql-createstatistics.html . We could drop the parenthetical "created via..." below if we do that.